validator/testcases/installrdf.py

```diff
@@ -30,16 +28,16 @@ def _test_rdf(err, install):
     """Wrapper for install.rdf testing to make unit testing so much
     easier."""
 
-    shouldnt_exist = ("hidden", )
+    shouldnt_exist = "hidden",
```

Comment:
I'm going to remove these changes for now. I'm not sure how they wound up in this PR.